infrastructure/template.yml

```diff
@@ -0,0 +1,155 @@
+  NginxImageTag:
+    Type: String
+    Description: NGINX Docker image tag
+
+Resources:
+  TaskDefinition:
+    Type: AWS::ECS::TaskDefinition
+    Properties:
+      Family: "canopas-website-dev-full-stack-task-definition"
+      ExecutionRoleArn: 
+        Fn::Sub: arn:aws:iam::${AWS::AccountId}:role/ecsTaskExecutionRole
+      NetworkMode: "bridge"
+      ContainerDefinitions:
+        - Name: "canopas-website-nginx"
+          Hostname: "canopas-website-nginx"
+          Memory: 128
+          Cpu: 128
+          Essential: true
+          Image: 
+            Fn::Sub: ${AWS::AccountId}.dkr.ecr.${AWS::Region}.amazonaws.com/canopas-website-nginx:${NginxImageTag}
+          PortMappings:
+            - ContainerPort: 80
+              HostPort: 80
+              Protocol: tcp
+          Links:
+            - canopas-website-frontend
+            - canopas-website-backend
+
+        - Name: "canopas-website-frontend"
+          Memory: 512
+          Cpu: 512
+          Essential: true
+          Image: 
+            Fn::Sub: ${AWS::AccountId}.dkr.ecr.${AWS::Region}.amazonaws.com/canopas-website-frontend:${ImageTag}
+          PortMappings:
+            - ContainerPort: 3080
+              HostPort: 3080
+              Protocol: tcp
+
+        - Name: "canopas-website-backend"
+          Memory: 256
+          Cpu: 256
+          Essential: true
+          Image: 
+            Fn::Sub: ${AWS::AccountId}.dkr.ecr.${AWS::Region}.amazonaws.com/canopas-website-backend:${BackendImageTag}
+          Environment:
+            - Name: ENV
+              Value: 
+                Fn::Sub: ${EnvName}
+            - Name: SENDER
+              Value: 
+                Fn::Sub: ${Sender}
+            - Name: JOBS_RECEIVER
+              Value: 
+                Fn::Sub: ${JobsReceiver}
+            - Name: CONTACT_RECEIVER
+              Value: 
+                Fn::Sub: ${ContactReceiver}
+            - Name: ACCESS_KEY_ID
+              Value: 
+                Fn::Sub: ${AccessKeyId}
+            - Name: SECRET_ACCESS_KEY
+              Value: 
+                Fn::Sub: ${SecretAccessKey}
+            - Name: REGION
+              Value: 
+                Fn::Sub: ${Region}
+            - Name: DB_USERNAME
+              Value: 
+                Fn::Sub: ${DbUsername}
+            - Name: DB_PASSWORD
+              Value: 
+                Fn::Sub: ${DbPassword}
+            - Name: DB_HOST
+              Value: 
+                Fn::Sub: ${DbHost}
+            - Name: DB_PORT
+              Value: 
+                Fn::Sub: ${DbPort}
+            - Name: DB_NAME
+              Value: 
+                Fn::Sub: ${DbName}
+            - Name: TZ
+              Value: UTC
+            - Name: GIN_MODE
+              value: release
+          PortMappings:
+            - ContainerPort: 8080
+              HostPort: 8080
+              Protocol: tcp
+
+  ECSService:
+    Type: AWS::ECS::Service
+    Properties:
+      ServiceName: "dev-full-stack"
```

Comment:
Do we need to take service name from argument or maybe append environment at end as the string will be different for prod and dev?

Reply:
As clustername is going to be different, maybe we can just name this canopas-website-full-stack?